### PR TITLE
feat: Show sessionData json on home page

### PR DESCRIPTION
**Pull request by merlinc**
- Add highlight.js to render JSON data in a styled way
- Show transformed sessionData as debugData in template

### What changed

Add a debug accordion onto the homepage, styled using [highlight js](https://highlightjs.org/)

### Screenshots

Screenshot from 2021-08-27 09-56-14

### Why did it change

It's difficult to understand and remember the data that has been collected. To help with debugging of the GPG45 profile matching, it's easier if we can see the raw data that has been used.

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: app/features/engine/index.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
 } from "./api";
 import Logger from "../../utils/logger";
 
-import { SessionData } from "../engine/api"
+import { SessionData } from "../engine/api";
```
Comment (merlinc) [inline app/features/engine/index.ts:26]: The changes in this file are unrelated, and are due to a previous commit not being linted/prettiered correctly.

File: app/features/ipv/home/view.njk [hunk 2]
```diff
@@ -294,6 +293,23 @@
             </dl>
             </div>
 
+
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/styles/stackoverflow-light.min.css">
```
Comment (merlinc) [inline app/features/ipv/home/view.njk:297]: This is not how we would add a stylesheet in production, however none of this code/design will end up in production - this is just to help us explore the problem space.

**Review by AmritSidhu — DISMISSED** [review]

**Review by AmritSidhu — APPROVED** [review]
